src/lib/chat/matrix-client.ts

```diff
@@ -736,6 +736,8 @@ export class MatrixClient implements IChatClient {
 
   async setUserAsModerator(roomId: string, user): Promise<void> {
     await this.waitForConnection();
+
+    this.lowerMinimumInviteAndKickLevels([this.matrix.getRoom(roomId)]);
```

Comment:
Needs an `await`? or, are we just assuming it'll eventually catch up and nobody will really notice?

Reply:
> are we just assuming it'll eventually catch up and nobody will really notice?

yeah i mean we don't process that event, & since this is a separate event from `"setUserAsMod"`, didn't feel like this call should be `awaited`. any reason why this should be `awaited`?

Reply:
The only reason I can think if is what I just asked. If we don't care if the user thinks it work but it isn't done then there's no need to await.

Eg: what if the user sets someone as a mod and then closes the browser prior to the "lower..." finishing? Does it actually complete or does it somehow cancel without finishing? If there's a chance it doesn't finish then the user will _think_ they've done what they wanted but it didn't actually do it.